store/types/store.go

```diff
@@ -194,9 +194,11 @@ type CommitMultiStore interface {
 	// ListeningEnabled returns if listening is enabled for the KVStore belonging the provided StoreKey
 	ListeningEnabled(key StoreKey) bool
 
-	// AddListeners adds WriteListeners for the KVStore belonging to the provided StoreKey
-	// It appends the listeners to a current set, if one already exists
-	AddListeners(key StoreKey, listeners []WriteListener)
+	// AddListeners adds q listener for the KVStore belonging to the provided StoreKey
+	AddListeners(keys []StoreKey)
```

Comment:
What's a `q listener`? It's a little confusing to not have an arg for the listeners when the function is named "Add Listeners". Would this be better named `EnableListening`?

Reply:
Ah it's a type. It needs to read `adds a listener` Old API use to take in a listener that you set up externally but it din't work well and was moved internally. AddListeners now attaches the internal listeners (`MemoryListener`) to the Store keys that you pass in.

I'll make the suggestion upstream as `EnableListening` is more fitting now that its attached internally but I would like to leave this in place and have the change come from upstream. What do you think?

Reply:
updated comment.